src/docker-outside-of-docker/install.sh

```diff
@@ -10,7 +10,7 @@
 DOCKER_VERSION="${VERSION:-"latest"}"
 USE_MOBY="${MOBY:-"true"}"
 MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
-DOCKER_DASH_COMPOSE_VERSION="${DOCKERDASHCOMPOSEVERSION:-"v1"}" # v1 or v2 or none
+DOCKER_DASH_COMPOSE_VERSION="${DOCKERDASHCOMPOSEVERSION:-"v2"}" # v1 or v2 or none
```

Comment:
Can you help bump the patch version for this Feature? Thanks! 

https://github.com/devcontainers/features/blob/a01cd8e70440dc72e96c72eaa07ab36a9e291b3a/src/docker-outside-of-docker/devcontainer-feature.json#L3